### ISSUE TITLE
Add Option to save Textures externaly by default

### ISSUE THREAD

**Issue by trashmaster9000** [ucupumar/ucupaint#115]
i'm currently making some kind of asset library, working on this big project quickly increased the Blender file size to over 1.2GB since this addon packs the images by default.

it would be nice to save the images externally by default and have some settings for image format / quality / path in addon settings

**Comment by ucupumar** [comment]
Hmm, I'm sorry but I think that's a very specific use case of ucupaint. I don't think I'll implement this myself anytime soon, feel free to implement this yourself or ask someone else who can do it for you. 🙏

**Comment by trashmaster9000** [comment]
would love to contribute to your awesome addon but sadly i can't program :/

**Comment by ucupumar** [comment]
Yeah, sorry. Maybe you can separate your library assets so it doesn't become that big.

**Comment by trashmaster9000** [comment]
don't worry!
i can always unpack all images from blender, i just have to remember to do that after a few new assets 😅
